packages/cli/src/utils/input.ts

```diff
@@ -46,18 +48,24 @@ function quoteArguments(args: string) {
   return args;
 }
 
-export function parseArg(input: string | string[], type: string): any {
+export function parseArg(input: string | string[], { type, components }: Pick<MethodArg, 'type' | 'components'>): any {
```

Comment:
Out of curiosity, in what cases is `input` an array?

Reply:
IIRC, `parseArray` does not parse just the top-level arrays, but it returns arrays of arrays (as many levels deep as needed). So, the `parseArg` recursive call in the case of arrays may be done with an array instead of a string.

And writing this I'm thinking that maybe arrays of tuples (or viceversa) may not work...